### PR TITLE
[EN-1388] Moves consul-template role to own repository

### PR DESCRIPTION
**Pull request by agirivera**
Story: http://jira.juwai.com/browse/EN-1388
Sub-task: http://jira.juwai.com/browse/EN-1399
- Adds galaxy tags
- No logical changes on the role from _ansible-playbooks_

How to test:
1. `git clone -b EN-1388 https://github.com/agirivera/ansible-role-consul-template.git`
2. `cd ansible-role-consul-template/tests`
3. `ansible-galaxy install -p roles -r requirements.yml`. _python27_ and _supervisor_ roles will fail for the moment, run the following instead:
   
   ```
   git clone -b EN-1388 https://github.com/agirivera/ansible-role-python27.git roles/juwai.python27
   git clone -b EN-1388 https://github.com/agirivera/ansible-role-supervisor.git roles/juwai.supervisor
   ```
4. `vagrant up`

/cc @juwai/platform @tjoelsson @imlazyone Please review. Thanks.

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  # validation errors on ansible galaxy
+  #
+  #- name: Amazon
+  #  versions:
+  #  - 2015.09
+
+  galaxy_tags: [amazon, centos6, "consul-template", python27, supervisor]
+
+dependencies:
+- role: supervisor
```
Comment (tjoelsson) [inline meta/main.yml]: juwai.supervisor

Comment (agirivera) [inline meta/main.yml]: Done. Thanks.

**Comment by tjoelsson** [comment]
👍

**Comment by stuartwan** [comment]
@tjoelsson did you test this already?

**Comment by tjoelsson** [comment]
I did

**Comment by stuartwan** [comment]
💅

**Pull request merged**